packages/strapi-plugin-settings-manager/admin/src/components/InputNumber/index.js

```diff
@@ -106,6 +106,9 @@ class InputNumber extends React.Component { // eslint-disable-line react/prefer-
     // set error class with override possibility
     const bootStrapClassDanger = !this.props.deactivateErrorHighlight && !isEmpty(this.state.errors) ? 'has-danger' : '';
     const placeholder = this.props.placeholder || `Change ${this.props.name} field`;
```

Comment:
Hard coded value? The words "Change" and "field" should be into a separate file? Again, I'm not sure about my comment on this point, just asking :)